tests/boulder-integration.sh

```diff
@@ -119,6 +119,8 @@ common revoke --cert-path "$root/conf/live/le1.wtf/cert.pem"
 common revoke --cert-path "$root/conf/live/le2.wtf/cert.pem" \
        --key-path "$root/conf/live/le2.wtf/privkey.pem"
 
+common register --deactivate
```

Comment:
This isn't actually deactivating the account because with the `--non-interactive`the defaults to `False` for deactivation.

Is there a way around using the default in `--non-interactive` mode? Should account deactivation not default to `False`?